### DISCUSSION TITLE
Steps to setup the Application locally

### DISCUSSION THREAD

**Discussion by ghousemohamed** [raxod502/riju#111] [Q&A] [answered]
@raxod502 Hey, I was looking through the source code and was wondering how to set it up locally on my machine, so that I can play around with it and understand the code running behind the application. Thank you for sharing and making this open source.

[EDIT] Started the server using `make dev`

**Comment by raxod502** [comment, marked as answer]
Hey @ghousemohamed! Thanks for pointing out that basic information on local setup was missing from the documentation. I've added some here: https://github.com/raxod502/riju/commit/24bac2f1e04004c793f335f99e19f37e2170df5e

Let me know if there's anything that would be helpful to append!
